webrtc-stats.html

```diff
@@ -1216,6 +1216,9 @@ <h3>
              unsigned long        framesEncoded;
              double               totalEncodeTime;
              double               averageRTCPInterval;
+             RTCQualityLimitationReason                 qualityLimitationReason;
```

Comment:
Should this have " = none" appended to make it clear that it's "none" by default?
Or should we make it explicitly "none" when none, so that we can tell that it's not supported when the value is missing?
Description should probably make this clear.

Reply:
It's in the description, "or "none" if not limited".